packages/presentation/src/PostMessageReleases.tsx

```diff
@@ -0,0 +1,33 @@
+import type {ClientPerspective} from '@sanity/client'
+import {memo, useEffect, type FC} from 'react'
+import {useReleases} from 'sanity'
+import type {VisualEditingConnection} from './types'
+
+export interface PostMessageReleasesProps {
+  comlink: VisualEditingConnection
+  perspective: ClientPerspective | `bundle.${string}`
+  bundlesPerspective: string[]
+}
+
+const PostMessageReleases: FC<PostMessageReleasesProps> = (props) => {
+  const {comlink, perspective, bundlesPerspective} = props
+
+  const releases = useReleases()
+
+  useEffect(() => {
+    const run = async () => {
+      comlink.post({
+        type: 'presentation/releases',
+        data: {
+          releases: releases.data,
+        },
+      })
+    }
+
+    run()
+  }, [comlink, perspective, bundlesPerspective])
```

Comment:
You will probably want here `releases.data` as a dependency ?

Reply:
oops sorry good catch 👍